tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -140,4 +140,154 @@ var _ = Describe("odo devfile catalog command tests", func() {
 		})
 	})
 
+	Context("odo devfile catalog command tests without kubeconfig", func() {
+		JustBeforeEach(func() {
+			helper.LocalKubeconfigSet("")
+		})
+
+		AfterEach(func() {
+			helper.LocalKubeconfigSet(context)
```

Comment:
Could you please let me know why we need to set `kubeconfig` in `AfterEach`. AFAIU this Context is meant for no kubeconfig, right?